005-trim/trim.sh

```diff
@@ -35,6 +35,7 @@ function doTrim()
     AdapterRemoval --file1 $fastq --file2 $fastq2 --output1 $out --output2 $out2 \
                    --threads $(nproc --all) \
                    --singleton $singletons --gzip --trimns --minlength 30 \
+                   --trim5p $toTrim --trim3p $toTrim \
```

Comment:
If you want to avoid the zero case, you could do this:

```sh
if [ $toTrim -eq 0 ]
then
    trimArgs=
else
    trimArgs="--trim5p $toTrim --trim3p $totrim"
fi
```

And then just put `$trimArgs` into the list of args you pass to `AdapterRemoval`.